### PR TITLE
update virtual cluster doc

### PR DESCRIPTION
**Pull request by xwzheng1020**
add a paragraph to describe virtual cluster

### CODE REVIEW

**Comment by xwzheng1020** [comment]
@scarlett2018

**Comment by xwzheng1020** [comment]
Screenshots are not so neccesary. It may make the doc too heavy. Users also can understand this feature when seeing the page.

File: docs/webportal/README.md [hunk 1]
```diff
@@ -115,6 +115,15 @@ Click the tab "Cluster View" to see the status of the whole cluster. Specificall
...
 * K8s Dashboard: The Kubernetes Dashboard.
 
+### Virtual cluster management
+
+OpenPAI Virtual Cluster is designed to run jobs as a shared, multi-tenant cluster in an operator-friendly manner while maximizing the throughput and the utilization of the cluster. Click the tab "Virtual Cluster" to  see virtual cluster's status and change virtual clusters for user and admin respectively. Specifically:
+
+* Add a new virtual cluster
+* Remove an obsolete virtual cluster
+* Increase or decrease virtual cluster's capacity
+* Change virtual cluster's availability
```
Comment (fanyangCS) [inline docs/webportal/README.md]: can you explain a bit more in the document about the meaning of vc availability?

Comment (xwzheng1020) [reply docs/webportal/README.md]: I also add some description about capacity.

**Pull request merged**